modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/manager/ExtensionErrorsRegistrantTestCase.java

```diff
@@ -96,7 +96,7 @@
   @Before
   public void setUp() {
     XmlDslModel.XmlDslModelBuilder builder = XmlDslModel.builder();
-    builder.setNamespace(EXTENSION_NAMESPACE);
+    builder.setPrefix(EXTENSION_NAMESPACE);
```

Comment:
not sure if you didn't see the commit [fixes after MGs review](https://github.com/mulesoft/mule/pull/4180/commits/01c58861231d7e49389197e63138a30452b2d9a2), but they weren't ignored